src/ray/rpc/rpc_chaos.cc

```diff
@@ -66,32 +74,34 @@ class RpcFailureManager {
       return RpcFailure::None;
     }
 
-    uint64_t &num_remaining_failures = iter->second;
-    if (num_remaining_failures == 0) {
+    auto &failable = iter->second;
+    if (failable.num_failures == 0) {
       return RpcFailure::None;
     }
 
-    std::uniform_int_distribution<int> dist(0, 3);
+    std::uniform_int_distribution<int> dist(0, 100);
```

Comment:
```suggestion
    std::uniform_int_distribution<int> dist(1, 100);
```
this is a bug when random_number is 0 and req_failure_prob is 0 as well.